homeassistant/components/roborock/time.py

```diff
@@ -79,6 +79,42 @@ class RoborockTimeDescription(TimeEntityDescription, RoborockTimeDescriptionMixi
         ),
         entity_category=EntityCategory.CONFIG,
     ),
+    RoborockTimeDescription(
```

Comment:
Perhaps neither the switch not the time entities should be enabled by default?

Reply:
Totally fine with this if that's what you think is better - but Whats your thought process for it?

Reply:
In the entity description set `entity_registry_enabled_default` to `False` for both the switch and the time entities and then the user can enable them if they want to use them.

Reply:
Ah sorry - I knew that part. I more so meant the reasoning for disabling it by default

Reply:
Let's say it's your idea if they are going to be used by many or not. 
There is no specific rule.

I'm asking as I can imagine these entities are not something everyone will use and thereby might "flood" ones dashboard with unnecessary entities/info. 

Was sort of thinking the same with the DND entities when you made those.

Reply:
That's fair - I will make that change sometime tomorrow and I'm going to try to improve documentation so that users know it exist when they need it